### PR TITLE
Fixes for Root v6-20-02. Temorary switch off the test of run_trac_its.C.

### PR DESCRIPTION
**Pull request by pzhristov**

### CODE REVIEW

File: macro/CMakeLists.txt [hunk 1]
```diff
@@ -359,18 +359,20 @@ o2_add_test_root_macro(run_trac_ca_its.C
...
-                                             O2::DataFormatsParameters
-                                             O2::DetectorsBase
-                                             O2::Field
-                                             O2::ITSBase
-                                             O2::ITSReconstruction
-                                             O2::ITStracking
-                                             O2::MathUtils
-                                             O2::SimulationDataFormat
-                       LABELS its)
+# PH: temporary switch off the test below until an issue in Detectors/ITSMFT/ITS/tracking/include/ITStracking/json.h
```
Comment (shahor02) [inline macro/CMakeLists.txt:362]: @pzhristov sorry if I missed something, what is the issue with the json.h ?

Comment (pzhristov) [reply macro/CMakeLists.txt:362]: When I use Root v6-20-02-alice3, the following issue appears in the test:
In file included from /home/phristov/alisw/O2/macro/run_trac_its.C:33:
In file included from /home/phristov/alisw/sw/SOURCES/O2/v1.2.0/0/Detectors/ITSMFT/ITS/tracking/include/ITStracking/IOUtils.h:26:
/home/phristov/alisw/sw/SOURCES/O2/v1.2.0/0/Detectors/ITSMFT/ITS/tracking/include/ITStracking/json.h:16211:13: error: no function template matches function template specialization 'swap'
inline void swap(nlohmann::json& j1,
            ^
/usr/include/c++/7/bits/basic_string.h:6246:5: note: candidate template ignored: could not match 'basic_string' against 'basic_json'
    swap(basic_string<_CharT, _Traits, _Alloc>& __lhs,
    ^
/usr/include/c++/7/array:294:5: note: candidate template ignored: could not match 'array' against 'basic_json'
    swap(array<_Tp, _Nm>& __one, array<_Tp, _Nm>& __two)
    ^
/usr/include/c++/7/array:302:5: note: candidate template ignored: could not match 'array' against 'basic_json'
    swap(array<_Tp, _Nm>&, array<_Tp, _Nm>&) = delete;
    ^

Comment (shahor02) [reply macro/CMakeLists.txt:362]: I see, another problem with cling, I guess again namespaces...
We can remove the macro from the tests, but this problem means that we will not be able to run the reco from the macro.
@mpuccio could you think of forward declaring the json and moving the header to the ``IOUtils.cxx`` ?

Comment (mpuccio) [reply macro/CMakeLists.txt:362]: Sure, I am going to open a PR

Comment (pzhristov) [reply macro/CMakeLists.txt:362]: Ok, I propse to merge this pull request, fix the macro and then reenable the test.

**Pull request merged**